doc/classes/Transform3D.xml

```diff
@@ -21,6 +21,7 @@
 			<return type="Transform3D" />
 			<description>
 				Constructs a [Transform3D] identical to the [constant IDENTITY].
+				[b]Note:[/b] In C#, this constructs a [Transform3D] with its [member origin] set to [constant Vector3.ZERO] and its [member basis] with all of its components set to [constant Vector3.ZERO].
```

Comment:
Yeah, it is more correct now but perhaps it's too verbose.
```suggestion
				[b]Note:[/b] In C#, this constructs a [Transform3D] with its [member origin] and the components of its [member basis] set to [constant Vector3.ZERO].
```
This may be more ambiguous but, given the other notes and the use of **Vector3.ZERO**, the user probably gets the picture.